system/ee/ExpressionEngine/Addons/channel/mod.channel.php

```diff
@@ -4056,15 +4072,19 @@ public function category_heading()
 
         // Is the category being specified by ID?
 
-        if (! preg_match("#(^|\/)C(\d+)#", $qstring, $match)) {
+        if (! preg_match("#(^|\/)C(\d+)#", $qstring, $match) AND ! ee()->TMPL->fetch_param('category_id'))
+        {
```

Comment:
According to PSR-12, opening brace should be on same line with `if`

Reply:
Jesus... I was pretty sure it was the other way! Fixing it right now!

Reply:
It is the other way for classes and functions. Easy to get confused!